avm/res/compute/virtual-machine/main.bicep

```
@@ -617,10 +617,10 @@ resource vm_autoShutdownConfiguration 'Microsoft.DevTestLab/schedules@2018-09-15
             notificationLocale: contains(autoShutdownConfig, 'notificationLocale')
               ? autoShutdownConfig.notificationLocale
               : 'en'
-            timeInMinutes: contains(autoShutdownConfig, 'notificationWebhookUrl')
+            webhookUrl: contains(autoShutdownConfig, 'notificationWebhookUrl')
```

If there a specific reason to renamr the original properties? If not I'd suggest to keep them all aligned. E.g., 
```suggestion
            webhookUrl: contains(autoShutdownConfig, 'webhookUrl')
```

---

@eriqua it is a sub property of notifications, thus the parent property before

---

If we follow that as a rule then I guess it should also be dailyRecurrenceTime and dailyRecurrenceTimeZoneId? Probably an overkill but I'll leave the decision to you as a module owner. I'll approve and keep this open for you to resolve if you want to merge as is.

---

@eriqua changed to dailyRecurrenceTime. The TimeZone is not part of the recurrence. The main reason for using the parent in the first place was, that status was already used by the parent, thus I needed a new name for the notification status